inst/EpiModel2.rmd

```diff
@@ -166,6 +176,7 @@ Here, we want to record the number of new infected - `nInf` -  at the current ti
 The `get` group of functions retrieve the value of an item on a sub-list of the `dat` objects, and are most often used when retrieving input parameters of `init`, `param` and `control` objects or for querying current attribute or epidemic tracker lists. As an example:
```

Comment:
suggestions for line 172: copy-edits

> Here, we want to record the number of new infected, `nInf`,  at the current time step, `at`. We do so using the `set_epi` accessor function and record the flow of susceptible to infected on to the master `dat` object. All `set` functions follow this naming scheme, with "set_" followed by the sub-list of interest. Note: the `set` functions return an updated `dat` object.

suggestions for line 176: copy-edit - been adding in Oxford commas 

```suggestion
The `get` group of functions retrieve the value of an item on a sub-list of the `dat` objects, and are most often used when retrieving input parameters of `init`, `param`, and `control` objects or for querying current attribute or epidemic tracker lists. As an example:
```